README.md

```diff
@@ -29,7 +29,7 @@ Check out the website: https://linkfree.eddiehub.io
 
 ## 👇 Prerequisites
 
-Before installation, please make sure you have already installed the following tools:
+Before Contributing or adding a new feature, please make sure you have already installed the following tools:
```

Comment:
Hmm, I'm not sure if we should capitalize `contributing` here.
What do you all think about this?


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
Yeah you are correct